src/main/webapp/app/delete-dialog/delete-dialog.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+    selector: 'jhi-delete-dialog',
+    templateUrl: './delete-dialog.component.html',
+})
+export class DeleteDialogComponent {
+    confirmExerciseName: string;
```

Comment:
I think we need a different name, as this dialog is also used for entities other than exercises.

Reply:
Changed it